src/Features/Core/Portable/LanguageServices/SymbolDisplayService/AbstractSymbolDisplayService.AbstractSymbolDescriptionBuilder.cs

```diff
@@ -151,6 +151,38 @@ protected SemanticModel GetSemanticModel(SyntaxTree tree)
 
                 AddOverloadCountPart(symbols);
                 FixAllAnonymousTypes(symbols[0]);
+                AddExceptions(symbols[0]);
+            }
+
+            private void AddExceptions(ISymbol symbol)
+            {
+                // All values returned by DocumentationComment.ExceptionTypes are normalized to the form:
+                //  T:Fully.Qualified.Name
+                // To turn these strings into ITypeSymbols, a quick sanity check is performed for the "T:"
+                // prefix before it is trimmed off and parsed.  All strings that don't parse are simply
+                // ignored
+                var exceptions =
+                    symbol.GetDocumentationComment().ExceptionTypes
+                    .Distinct()
+                    .Where(e => e.StartsWith("T:"))
+                    .Select(e => e.Substring(2))
+                    .OrderBy(e => e)
+                    .Select(_semanticModel.Compilation.GetTypeByMetadataName)
+                    .WhereNotNull();
```

Comment:
One more thing, you're calculating this twice for the first exception - once for the Any call, and then once again for the foreach.